### PR TITLE
Ai sat maint fix

### LINKED ISSUE #58546: icebox Sat maint doesn't protect against rad storms
Reporting client version: 513.1536

## Round ID:
irrelevant

## Testmerges:
Manuel
- [[DNM] doubles all turf decals to test whether they actually affect maptick at all under live conditions](https://www.github.com/tgstation/tgstation/pull/58496)

## Reproduction:
1. have a rad storm accord
2. attempt to hide from it in AI sat maint
3. get irradiated

### PR DESCRIPTION
**Pull request by RandomGamer123**
## About The Pull Request
Makes AI Satellite areas that are "maintenance" (those that previously had a door that said "AI Satellite Maintenance" or similar connecting to them) actually radproof by adding a new /aisat/maint subarea as well as adding /ai_monitored/command/storage/satellite to the radstorm proof area list.

Note: (First time messing with maps, probably messed something up somewhere)

## Why It's Good For The Game
Fixes #58546 , if the airlock literally says "AI Satellite Maintenance" on it you would expect it to be radproof

## Changelog
:cl:
fix: Maintenance sections of AI satellites are now actually radiation storm proof
/:cl:

### CODE REVIEW

**Comment by RandomGamer123** [comment]
wait i merged from master why did oh tgm i think

**Comment by RandomGamer123** [comment]
wait no uhh why are there 48k lines diff something is terribly wrong but i have no idea what

**Comment by RandomGamer123** [comment]
nope rebasing didn't help

**Comment by JohnFulpWillard** [comment]
> 
> 
> wait no uhh why are there 48k lines diff something is terribly wrong but i have no idea what

map files are really big

**Comment by RandomGamer123** [comment]
> > wait no uhh why are there 48k lines diff something is terribly wrong but i have no idea what
> 
> map files are really big

so everything is fine then hopefully

**Comment by zxaber** [comment]
Did you run mapmerger? Because if not, that's probably why you have 48k lines of change.

**Comment by RandomGamer123** [comment]
If you mean mapmerge2\fixup.py then yes

**Comment by RandomGamer123** [comment]
merging all commits also didn't help, but mapdiffbot says that everything is fine, guess that's just the case

**Comment by RandomGamer123** [comment]
> Did you run mapmerger? Because if not, that's probably why you have 48k lines of change.

actually i think that might be the issue after checking as it seems like the maps aren't correctly merged, but uh how do i fix that

**Comment by RandomGamer123** [comment]
is there a way to run mapmerge manually

**Comment by zxaber** [comment]
You'll want the `Run Before Committing.bat` script.

**Comment by RandomGamer123** [comment]
@zxaber thank you so much

File: html/changelogs/AutoChangeLog-pr-59099.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author: "Timberpoes"
```
Comment (Maurukas) [inline html/changelogs/AutoChangeLog-pr-59099.yml:1]: This is a changelog file. Please remove this from the PR and we should be good to go.

**Review by Maurukas — CHANGES_REQUESTED** [review]

File: html/changelogs/AutoChangeLog-pr-59099.yml (hunk 1 above)
Comment (RandomGamer123) [reply html/changelogs/AutoChangeLog-pr-59099.yml]: Oops fixed.

**Review by Maurukas — APPROVED** [review]

**Pull request merged**